src/pages/dashboard/components/EpisodeDetails.tsx

```diff
@@ -66,10 +66,38 @@ function EpisodeDetails({ episode, isInCollection = false, showDate = false }: P
     return `${((resumePosition.asMilliseconds() / duration.asMilliseconds()) * 100).toFixed(2)}%`;
   }, [episode.Duration, episode.ResumePosition]);
 
+  const anidbEpisodeAlias = (type: EpisodeTypeEnum, epTitle: string) => {
+    // TODO: This is a hack, we need update backend to make the EpisodeTypeEnum make more sense.
+    if (type === EpisodeTypeEnum.ThemeSong) {
+      if (epTitle.includes('Opening')) return 'OP';
+      if (epTitle.includes('Ending')) return 'ED';
+      return '';
+    }
+
+    // Prefixes for episode types base on https://wiki.anidb.net/Content:Episodes
+    switch (type) {
+      case EpisodeTypeEnum.OpeningSong:
+        return 'OP';
+      case EpisodeTypeEnum.EndingSong:
+        return 'ED';
+      case EpisodeTypeEnum.Special:
+      case EpisodeTypeEnum.Extra:
```

Comment:
Extra is not used…  yet.

Reply:
I think I just use the prefixes base on AniDb for `Extra` for now